internal/cmd/agent/deployer/monitor/updatestatus.go

```diff
@@ -101,14 +106,16 @@ func (m *Monitor) UpdateStatus(ctx context.Context, bd *fleet.BundleDeployment,
 
 		return origStatus, err
 	}
+
 	status := bd.Status
+	status.SyncGeneration = &bd.Spec.Options.ForceSyncGeneration
 
 	readyError := readyError(status)
 	Cond(fleet.BundleDeploymentConditionReady).SetError(&status, "", readyError)
-
-	status.SyncGeneration = &bd.Spec.Options.ForceSyncGeneration
 	if readyError != nil {
-		logger.Info("Status not ready", "error", readyError)
+		logger.Info("Status not ready according to nonModified and nonReady", "nonModified", status.NonModified, "nonReady", status.NonReadyStatus)
+	} else {
+		logger.V(1).Info("Status ready, Deployed condition set to true")
```

Comment:
If this refers to the operation run on L114, isn't that the `Ready` condition?

Reply:
Ah, I misread that for `BundleDeploymentConditionDeployed`.